src/pages/p2p/components/_dp2p.js

```diff
@@ -111,9 +121,25 @@ const DP2P = ({ P2P, reverse, two_title }) => {
             <StyledContainer>
                 <StyledText>
                     {localize(
-                        'DP2P is Deriv’s peer-to-peer deposit and withdrawal service. It’s where you can get money in and out of your Deriv account via exchanges with fellow traders.',
+                        'Deriv P2P is Deriv’s peer-to-peer deposit and withdrawal service that offers an easy way to get money in and out of your Deriv account. Connect with fellow traders and transfer money in minutes.',
                     )}
                 </StyledText>
+
+                <VideoText>
+                    {localize(
+                        'Find out how Deriv P2P works:',
+                    )}
+                </VideoText>
+                <iframe
+                  width="560"
+                  height="315"
+                  src="https://www.youtube.com/embed/zf9flqE94Ek"
+                  title="YouTube video player"
+                  frameBorder="0"
+                  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                  allowFullScreen>
+                </iframe>
```

Comment:
You're gonna need to extend this and style it for mobile for as low as 320px as well. 🙏

Reply:
oh yeah sure, let me do that